lib/cc/engine/analyzers/analyzer_base.rb

```diff
@@ -51,7 +51,7 @@ def files
         end
 
         def filters
-          engine_config.filters_for(language)
+          engine_config.filters_for(language) | default_filters
```

Comment:
I don't know if we want the union of these two sets: a user overriding the default might reasonably want to *not* use one or more of the default filters, yeah?

Reply:
I can't imagine a way for a user to override the default, without making it a `default_filters` OR `.codeclimate.yml` backed filters. (since it's an array, instead of hash where we could do some sort of `merge!`)

I'd propose going with this implementation (which seems simplest and desirable for most users), and then changing if we get feedback, because:

I can't imagine users _wanting_ to get `import` and `package` statements flagged. 

I _could_ imagine someone adding a filter for a different node type, then being surprised that it triggered PackageDeclarations and ImportDeclarations to stop being ignored.

The (IMO less desirable) alternative would be to use an `OR` and update documentation, see if that generates confusion.

WDYT?

Reply:
I think this behavior sounds fine as a starting point. Any user-supplied filters would be added to our default list, which we've determined safe to ignore for all users.

If we made it an override, users would have to pick out filters from the default set that they'd like to ignore, which seems undesirable.